boards/yarm/Makefile.include

```diff
@@ -1,5 +1,3 @@
-include $(RIOTMAKE)/boards/sam0.inc.mk
```

Comment:
are you sure you can remove this?

Reply:
Hmm, you are right this board is flashed using edbg so this include in needed.

Reply:
reverted. I checked and there's indeed a JTAG on the board, so can be programmed with an external flasher using openocd for example.